charts/datadog/templates/kpi-telemetry-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kpi-telemetry-configmap
```

Comment:
I'm not sure it is good to have a fix name here. because we have some use case (like windows install that require to deploy/install twice this chart in the same namespace)

Also can we create this configmap only if APM SSI is enabled?

Reply:
- Parametrized the Configmap name with `.Release.name`
- We need this env variables to get KPIs for all of manual instrumentation, local library injection and SSI. Thus we cannot condition configmap on SSI enablement.

Reply:
Ok but maybe at least when the admission controller is enabled

Reply:
Sorry, I was too quick merging this. This is not in parity with the Operator, which adds these envs only if APM feature is enabled. If we add AC check here, we should update Operator code, otherwise we add APM check here.

Reply:
> This is not in parity with the Operator, which adds these envs only if APM feature is enabled
This is the [Operator code](https://github.com/DataDog/datadog-operator/blob/5ef94b21b94a17254efadfac73ee189eba7b0c41/controllers/datadogagent/component/clusteragent/default.go#L148-L159) that sets these env variables. I believe we always set these env variables and not only when APM is enabled.

Reply:
I was referring to trace agent, which is not run when APM is disabled so behavior should match. Sorry for confusion.